internal/renderer/helm.go

```diff
@@ -207,21 +215,41 @@ func makeChart(appname, targetDir string) error {
 	return ioutil.WriteFile(filepath.Join(targetDir, "Chart.yaml"), hmetadata, 0644)
 }
 
-func helmRender(appname string, targetDir string, composeFiles []string, settingsFile []string, env map[string]string) error {
+func helmRender(appname string, targetDir string, composeFiles []string, settingsFile []string, env map[string]string, beta1 bool) error {
 	rendered, err := Render(appname, composeFiles, settingsFile, env)
 	if err != nil {
 		return err
 	}
-	stackSpec := conversion.FromComposeConfig(rendered)
-	stack := v1beta2.Stack{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "stacks.compose.docker.com",
-			APIVersion: "v1beta2",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name: internal.AppNameFromDir(appname),
-		},
-		Spec: stackSpec,
+	var stack interface{}
+	if !beta1 {
```

Comment:
We should use a `switch` on the `stackVersion` string (and have a default that errors out)